packages/account-management/src/frontend/Registration/Registration.tsx

```diff
@@ -59,8 +59,14 @@ export const Registration: React.FC = () => {
   }, []);
 
   useEffect(() => {
-    if (email !== '') setEmailValid(emailTest.test(email || ''));
-    if (email === '') setEmailValid(true);
+    if (email !== '') {
+      setEmailValid(emailTest.test(email || ''));
+      setAlreadyExists(false);
+    }
+    if (!email || email === '') {
```

Comment:
Will this result in the user being trying to create an account with a blank password field?

Reply:
No, we don't actually make a post to the API unless all the fields have been entered (see line 83). This is just about making sure that the error on the email field isn't shown on initially loading the page (the initial condition on this check was `email === ''` which was evaluating false on initial page load), and getting rid of the duplicate email address error when changing the email field. Without this, if you were to use a duplicate email and common password, submit the form, then change the email address and resubmit, the initial error message about the email being duplicated would still be hanging around despite this not being the case any more.

This does remind me that I thought I'd made a similar change on the password field which no longer seems to be there after cherry picking my changes over to the latest master so I'm just going to double check that that